manifests/c/chrisant996/Clink/1.4.18.792517/chrisant996.Clink.installer.yaml

```
@@ -1,27 +1,26 @@
-# Created using wingetcreate 1.2.5.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
-
-PackageIdentifier: chrisant996.Clink
-PackageVersion: 1.4.18
-MinimumOSVersion: 10.0.0.0
-Scope: machine
-InstallModes:
-- interactive
-- silent
-UpgradeBehavior: install
-ProductCode: clink_1.4.16.b8efc4
-Installers:
-- Architecture: x64
-  InstallerType: nullsoft
-  InstallerUrl: https://github.com/chrisant996/clink/releases/download/v1.4.18/clink.1.4.18.792517_setup.exe
-  InstallerSha256: 9782F3912B3B311ABF48663D4CA2BE04E7D2BA9EC1E06691157D0B9B6281A3B6
-  ProductCode: clink.1.4.18.792517
-  ReleaseDate: 2023-02-12
-- Architecture: x86
-  InstallerType: nullsoft
-  InstallerUrl: https://github.com/chrisant996/clink/releases/download/v1.4.18/clink.1.4.18.792517_setup.exe
-  InstallerSha256: 9782F3912B3B311ABF48663D4CA2BE04E7D2BA9EC1E06691157D0B9B6281A3B6
-  ProductCode: clink.1.4.18.792517
-ManifestType: installer
-ManifestVersion: 1.4.0
-ReleaseDate: 2023-02-12
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: chrisant996.Clink
+PackageVersion: 1.4.18.792517
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/chrisant996/clink/releases/download/v1.4.18/clink.1.4.18.792517_setup.exe
+  InstallerSha256: 9782F3912B3B311ABF48663D4CA2BE04E7D2BA9EC1E06691157D0B9B6281A3B6
+  ProductCode: clink.1.4.18.792517
```

I'm not sure why you are removing ProductCode for this package in other PRs 👀I'm getting this as ProductCode
```suggestion
  ProductCode: clink_1.4.18.792517
```

---

@mdanish-kh Thanks for correcting it here; I'm removing it in other PR's because it's at both the installer and root level in the other versions, and is mismatched in some cases